CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ cmake_minimum_required(VERSION 3.0)
 set(CMAKE_CXX_STANDARD 14)
 
 project(OpenCL-SDK
-  VERSION 1.0
+  VERSION 2022.03.09
```

Comment:
Presumably this needs to change when we do a release?  Does anything else need to change?

I think it would be useful to have a short document that describes the process for doing a release.

Reply:
I've added such a document. Please check if it's in line with what you had in mind.

Reply:
This is perfect - thanks!